pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -92,3 +115,88 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
+
+type ScaleREST struct {
+	registry statefulset.Registry
+}
+
+// ScaleREST implements Patcher
+var _ = rest.Patcher(&ScaleREST{})
+
+// New creates a new Scale object
+func (r *ScaleREST) New() runtime.Object {
+	return &extensions.Scale{}
+}
+
+func (r *ScaleREST) Get(ctx genericapirequest.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	ss, err := r.registry.GetStatefulSet(ctx, name, options)
+	if err != nil {
+		return nil, errors.NewNotFound(extensions.Resource("statefulsets/scale"), name)
```

Comment:
this should only return a NotFound error if we got a NotFound error looking up the stateful set, right?